modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -18,6 +18,8 @@ for a presentation on Tang.
 If you miss this step, the second boot will fail.
 For example, to configure DHCP networking, identify `ip=dhcp`
 or set static networking when you add parameters to the kernel command line.
+For both DHCP and static networking, you also need to provide the `rd.neednet=1`
```

Comment:
@chrisnegus, will you PTAL?

Reply:
I didn't know this. we should confirm with @darkmuggle

Reply:
@darkmuggle can you confirm for @chrisnegus?

Reply:
This addition is implicitly confirmed by @darkmuggle in this comment on the linked BZ - https://bugzilla.redhat.com/show_bug.cgi?id=1819215#c5